react/week3/src/components/Todo.js

```diff
@@ -0,0 +1,63 @@
+import React , { useState } from 'react'
+
+function Todo({ todo, index, completeTodo, removeTodo, updateTodo }) {
+ 
+  const [editTask, seteditTask] = useState(false);
+  const [state, setState] = useState({newText: todo.text, newDeadline:todo.deadline});

```

Comment:
`state` is a very generic name. You should be more descriptive here. You should call this something like:
```suggestion
  const [todo, setTodo] = useState({newText: todo.text, newDeadline:todo.deadline});

```